torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1705,7 +1705,8 @@ def _make_tensor_helper(shape, low=None, high=None):
            # BFloat16 support on CUDA requires CUDA 11 and SM53
            dtypesIfCUDA=floating_types_and(torch.float16, torch.complex64, torch.complex128,
                                            *[torch.bfloat16] if CUDA11OrLater else []),
-           dtypesIfROCM=floating_types_and(torch.half),
+           dtypesIfROCM=floating_types_and(torch.float16, torch.complex64, torch.complex128,
+                                           *[torch.bfloat16] if CUDA11OrLater else []),
```

Comment:
```suggestion
           dtypesIfROCM=floating_types_and(torch.float16, torch.complex64, torch.complex128),
```
Let's leave out bfloat16 for now.  The `CUDA11OrLater` condition will always be false for HIP; removing it makes it more clear.

Reply:
New suggestion coming.